Hyperlint Automation: Meta Description Fixes (#20462)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/identity/idp-integration/pingone-saml.mdx

---
pcx_content_type: how-to
title: PingOne (SAML)

> description: Learn how to integrate PingOne as a SAML identity provider with Cloudflare Access. Follow step-by-step instructions for seamless SSO setup.

Reviewer: ```suggestion
description: Learn how to integrate PingOne as a SAML identity provider with Cloudflare Zero Trust.
```
